### PR TITLE
Room microservice

### PR DESCRIPTION
**Pull request by sylwiapytko**
Create booking, get booking info + talk to User Microservice Endpoints

### CODE REVIEW

**Comment by sylwiapytko** [comment]
https://www.getpostman.com/collections/3c1b9a61a8633c60f33f postman collection to test

File: conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/ConferenceRoom.java [hunk 1]
```diff
@@ -1,35 +1,63 @@
...
+
+    public int getFloor() {
+        return floor;
+    }
+
+    public void setFloor(int floor) {
+        this.floor = floor;
     }
 
     public String getName() {
```
Comment (nowpawel96) [inline conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/ConferenceRoom.java]: That can be changes to lombok annotations

Comment (nowpawel96) [reply conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/ConferenceRoom.java]: Be careful with hashcode and toString and do not use data annotation

File: conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/responseentitystructure/RoomBookingInfo.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.bestgroup.conferenceroomservice.responseentitystructure;
+
+import com.bestgroup.conferenceroomservice.conferenceroombooing.RoomBooking;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```
Comment (nowpawel96) [inline conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/responseentitystructure/RoomBookingInfo.java]: Change it to discussed annotations

File: conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/conferenceroombooing/RoomBooking.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+@Entity
+@Data
+public class RoomBooking {
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private int roomBookingId;
+
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name="roomId")
```
Comment (nowpawel96) [inline conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/conferenceroombooing/RoomBooking.java]: JoinColumn should be specifiedn on OneToMany side by mappedby=

File: conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/responseentitystructure/RoomBookingInfo.java (hunk 2 above)
Comment (sylwiapytko) [reply conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/responseentitystructure/RoomBookingInfo.java]: This is not an Entity

Comment (nowpawel96) [reply conference-room-service/src/main/java/com/bestgroup/conferenceroomservice/responseentitystructure/RoomBookingInfo.java]: ok seems legit

**Pull request merged**